src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  switch (true) {
+    case (days >= 3 && days < 7):
+      return days * 40 - 20;
```

Comment:
```suggestion
      return days * carRent - discountForThreeDays;
```
Checklist #1, you should use variables and give them descriptive names.

Reply:
+1, let's fix that